extensions/wikia/PageHeader/templates/PageHeaderController_categories.php

```diff
@@ -0,0 +1,29 @@
+<?php if ( $categories->hasVisibleCategories() ): ?>
+	<div class="page-header__categories">
+		<span class="page-header__categories-in" data-tracking="categories-top-in"><?= $categories->inCategoriesText ?>:</span>
+		<div class="page-header__categories-links">
+			<?= $categories->visibleCategoriesHTML ?>
+			<?php if ( $categories->hasMoreCategories() ): ?>
+				<div class="wds-dropdown page-header__categories-dropdown">
+					<a class="push-dropdown-down wds-dropdown__toggle" data-tracking="categories-more"><?= $categories->moreCategoriesText ?></a>
```

Comment:
do we need this class `push-dropdown-down`? it's for DS but not in style-guide styles